install.sh

```diff
@@ -208,6 +246,10 @@ if [ -w "$agentPath/tmp" ] ; then
 else
   error_exit "Temporary Folder $agentPath/tmp in agent directory is not writable"
 fi
+cd "$appRoot"
+echo "Your default path to php - $phpPath (to override you might set AGENT_INSTALLER_PHP as env variable before running installer)"
+checkJwt $sandboxEnv "$phpPath"
```

Comment:
move cd and echo inside _checkJwt_ 

```
function checkJwt()
{
  local appPath="$3"
}
```

OR
Run cd in subshell to not change the context of parent shell. 
```
( cd "$appRoot" ; checkJwt $sandboxEnv "$phpPath" )
```